packages/web-components/src/components/tabs-extended/tab.ts

```diff
@@ -30,7 +31,7 @@ class C4DTab extends StableSelectorMixin(CDSTab) {
    * An event that fires whenever the tab's selected state changes.
    */
   static get eventTabSelected() {
```

Comment:
Note that since we're defining this event name in `defs.ts`, we could _probably_ get rid of this getter. I'm intentionally leaving it here though on the off chance an adopter is referencing it.

Reply:
From a purely theoretical standpoint, I like having the getter. It allows extending classes to change the event name.